Build error

https://github.com/eoscostarica/guias.eoscostarica.io/runs/7633652093?check_suite_focus=true
```
[success] [webpackbar] Client: Compiled with some errors in 38.69s


Error:  Client bundle compiled with errors therefore further build is impossible.
Syntax error: /home/runner/work/guias.eoscostarica.io/guias.eoscostarica.io/src/css/homepage.css Unknown word (602:1)
  600 |      }
  601 |  }
> 602 | =======
      | ^
  603 | @import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");
  604 | 
Syntax error: /home/runner/work/guias.eoscostarica.io/guias.eoscostarica.io/src/css/homepage.css Unknown word (602:1)
  600 |      }
  601 |  }
> 602 | =======
      | ^
  603 | @import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");
  604 | 
-- inner error --
Syntax error: /home/runner/work/guias.eoscostarica.io/guias.eoscostarica.io/src/css/homepage.css Unknown word (602:1)

  600 |      }
  601 |  }
> 602 | =======
      | ^
  603 | @import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");
  604 | 
Generated code for /home/runner/work/guias.eoscostarica.io/guias.eoscostarica.io/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[6].use[1]!/home/runner/work/guias.eoscostarica.io/guias.eoscostarica.io/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[6].use[2]!/home/runner/work/guias.eoscostarica.io/guias.eoscostarica.io/src/css/homepage.css
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -14,8 +14,8 @@
     "serve": "docusaurus serve"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-rc.1",
-    "@docusaurus/preset-classic": "^2.0.0-rc.1",
+    "@docusaurus/core": "^2.0.1",
+    "@docusaurus/preset-classic": "^2.0.1",
     "@material-ui/core": "^4.12.4",
     "@mdx-js/react": "^1.6.22",
     "bootstrap": "^5.2.0",

--- a/src/css/homepage.css
+++ b/src/css/homepage.css
@@ -599,7 +599,6 @@
          margin-bottom: 25px;
      }
  }
-=======
 @import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");
 
 body {

--- a/src/theme/Footer/index.js
+++ b/src/theme/Footer/index.js
@@ -47,7 +47,7 @@
           <Grid container spacing={2}>
             <Grid container >
               <Grid item xs={12} sm={3}>
-               <Box align='center'>
+               <Box align='center' className={styles.footerLogo}>
                  <a href={logo.href}>
                    <FooterLogo alt={logo.alt} url={logoUrl} />
                  </a>

--- a/src/theme/Footer/index.js
+++ b/src/theme/Footer/index.js
@@ -60,7 +60,7 @@
                     <Grid item xs>
                      <Box className={styles.borderleft}>
                        <Box>
-                       <h4>Guías Generales</h4>
+                       <h4>EOS Costa Rica</h4>
                        </Box>
                        <Box>
                          <Link href="https://guide.eoscostarica.io/docs/engineering-culture" color="inherit">

--- a/src/theme/Footer/index.js
+++ b/src/theme/Footer/index.js
@@ -42,33 +42,26 @@
    }, [colorMode]);
  
    return (
-     <Box
-       borderTop={2}
-       borderColor="#2f80ed"
-       borderRadius="1"
-       px={{ xs: 3, sm: 4 }}
-       py={{ xs: 5, sm: 4 }}
-       className={clsx('styles.footerLigth', {
-         'styles.footerDark': footer.style === 'dark'
-       })}
-     >
+     <Box className={clsx('styles.footerLight', styles.bordertop, {'styles.footerDark': footer.style === 'dark'})}>
         <Container maxWidth="xl">
           <Grid container spacing={2}>
             <Grid container >
               <Grid item xs={12} sm={3}>
-               <Box align='center' >
+               <Box align='center'>
                  <a href={logo.href}>
                    <FooterLogo alt={logo.alt} url={logoUrl} />
                  </a>
                  {copyright ? <div>Copyright &reg; {new Date().getFullYear()} EOS Costa Rica</div> : null}
                 </Box>
              </Grid>
-              <Grid item xs={12} sm={6}>
-               <Box border={1} borderTop={0} borderBottom={0} borderColor={'#BDBDBD'} paddingLeft={20}>
+              <Grid item xs>
+               <Box className={styles.lineas}>
                  <Grid container >
-                    <Grid item xs={12} sm={4} >
-                     <Box fontSize={18} >
-                       <Box fontWeight={800} >EOS Costa Rica</Box>
+                    <Grid item xs>
+                     <Box className={styles.borderleft}>
+                       <Box>
+                       <h4>Guías Generales</h4>
+                       </Box>
                        <Box>
                          <Link href="https://guide.eoscostarica.io/docs/engineering-culture" color="inherit">
                           Cultura Ingenieril
@@ -86,9 +79,11 @@
                        </Box>
                      </Box>
                    </Grid>
-                   <Grid item xs={12} sm={4}>
-                     <Box fontSize={18}>
-                       <Box fontWeight={800}>Comunidad</Box>
+                   <Grid item xs>
+                     <Box className={styles.bordermid}>
+                       <Box>
+                        <h4>Comunidad</h4>
+                        </Box>
                        <Box>
                          <Link href="https://www.meetup.com/es/EOS-Costa-Rica/" color="inherit">
                            Meetup
@@ -106,9 +101,11 @@
                        </Box>
                      </Box>
                    </Grid>
-                   <Grid item xs={12} sm={4}>
-                     <Box fontSize={18}>
-                       <Box fontWeight={800}>Más</Box>
+                   <Grid item xs>
+                     <Box className={styles.borderRight}>
+                       <Box>
+                       <h4>Más</h4>
+                       </Box>
                        <Box>
                          <Link href="https://www.linkedin.com/company/eoscostarica/" color="inherit">
                            Linkedin
@@ -127,31 +124,31 @@
                      </Box>
                    </Grid>
                   </Grid>
-               </Box>
-             </Grid>
+                </Box>
+             </Grid>             
               <Grid item xs={12} sm={3}>
-               <div className={styles.contentSocial}>
-                 <Grid container spacing={1} >
-                   <Grid item xs={12} sm={4} align='right'> 
+               <Box className={styles.contentSocial}>
+                 <Grid container spacing={0} >
+                   <Grid item xs={2} sm={4} > 
                      <AiFillGithub size={40} onClick={()=> window.open("https://github.com/eoscostarica/")}/>
                    </Grid>
-                   <Grid item xs={12} sm={4} align='center'>
+                   <Grid item xs={2} sm={4} >
                      <AiFillLinkedin size={40} onClick={()=> window.open("https://www.linkedin.com/company/eoscostarica/")}/>
                    </Grid>
-                   <Grid item xs={12} sm={4} align='left'>
+                   <Grid item xs={2} sm={4} >
                      <AiOutlineTwitter size={40} onClick={()=> window.open("https://twitter.com/eoscostarica")} />
                    </Grid>
-                   <Grid item xs={12} sm={4} align='right'>
+                   <Grid item xs={2} sm={4} >
                      <BsMedium size={40} onClick={()=> window.open("https://eoscostarica.medium.com/")}/>
                    </Grid>
-                   <Grid item xs={12} sm={4} align='center'>
+                   <Grid item xs={2} sm={4} >
                      <AiOutlineInstagram size={40} onClick={()=> window.open("https://www.instagram.com/eoscostarica/")}/>
                    </Grid>
-                   <Grid item xs={12} sm={4} align='left'>
+                   <Grid item xs={2} sm={4} >
                      <AiFillYoutube size={40} onClick={()=> window.open("https://www.youtube.com/c/EOSCostaRica")}/>
                    </Grid>
                  </Grid>
-               </div>
+               </Box>
              </Grid>
             </Grid>
           </Grid>
@@ -159,5 +156,4 @@
      </Box>
    );
  }
- 
  export default Footer;

--- a/src/theme/Footer/styles.module.css
+++ b/src/theme/Footer/styles.module.css
@@ -81,4 +81,7 @@ h4 {
   border-top: 2px solid #2f80ed;
 }
 
+.footerLogo{
+  margin-top: 10px;
+}
 

--- a/src/theme/Footer/styles.module.css
+++ b/src/theme/Footer/styles.module.css
@@ -14,19 +14,73 @@
   opacity: 1;
 }
 
-.footerLigth{
-  background-color: #f2f2f2;
+.footerLight{
+  background-color:#f2f2f2 ;
 }
 
 .footerDark{
-  background-color: #333;
+  background-color: rgb(31, 27, 41);
 }
 
 .contentSocial{
+  margin-top: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: right;
+  flex-direction: column;
+  text-align: center;
+}
+
+
+h4 {
+  font-family: "Roboto", sans-serif;
+  font-weight: bold;
+  font-size: 20;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.25px;
+}
+
+.borderleft{
+  display: grid;
+  justify-content: center;
   margin-top: 25px;
+  font-size:14px;
+  border-left: 1px solid;
+  border-color: #bdbdbd;
+  line-height: 2.0;
+  margin-bottom: 25px;
 }
 
-.test{
-  border-top: 10px solid blue;
 
+.textBox{
+  display: grid;
+  justify-content: center;
 }
+
+.bordermid{
+  display: grid;
+  justify-content: center;
+  font-size:14px;
+  margin-top: 25px;
+  line-height: 2.0;
+  margin-bottom: 25px;
+}
+
+.borderRight{
+  display: grid;
+  justify-content: center;
+  font-size:14px;
+  margin-top: 25px;
+  border-right: 1px solid;
+  border-color: #bdbdbd;
+  line-height: 2.0;
+  margin-bottom: 25px;
+}
+
+.bordertop{
+  border-top: 2px solid #2f80ed;
+}
+
+

--- a/src/theme/Footer/styles.module.css
+++ b/src/theme/Footer/styles.module.css
@@ -31,7 +31,6 @@
   text-align: center;
 }
 
-
 h4 {
   font-family: "Roboto", sans-serif;
   font-weight: bold;
@@ -52,7 +51,6 @@ h4 {
   line-height: 2.0;
   margin-bottom: 25px;
 }
-
 
 .textBox{
   display: grid;

